engines/tika/src/main/java/org/alfresco/transform/tika/metadata/extractors/IPTCMetadataExtractor.java

```diff
@@ -55,6 +56,12 @@ public class IPTCMetadataExtractor extends AbstractTikaMetadataExtractorEmbeddor
     private static final Pattern YEAR_IPTC = Pattern.compile("(\\d{4}[:|-]\\d{2}[:|-]\\d{2})");
 
     private ExifToolParser parser;
+    private RuntimeExec exifRuntimeExec;
+
+    public IPTCMetadataExtractor(RuntimeExec exifRuntimeExec) {

```

Comment:
The `IPTCMetadataExtractorTest` uses this constructor and since it's the only constructor, think we've covered.
BTW, the parameter can be null (`ExifToolParser` will deal with nulls).

Reply:
Now the other constructor has been removed and the test class changed to use this one, I think we are fine